src/pages/docs/tutorials/tutorial-hello-enact/basics/index.md

```diff
@@ -56,13 +56,13 @@ The first step is to `import` our dependencies for this component. The [`import`
 ```js
 import React from 'react';
 ```
-We only have a single import right now but it's a very important one. With Enact, we've decided to build on top of the very popular [React library](https://facebook.github.io/react) from Facebook. The `react` module provides the tools necessary to create and compose components. Rendering is handled by another module, `react-dom`, which will be discussed [later](#rendering-the-app).
+We only have a single import right now but it's a very important one. With Enact, we've decided to build on top of the very popular [React library](https://reactjs.org) from Facebook. The `react` module provides the tools necessary to create and compose components. Rendering is handled by another module, `react-dom`, which will be discussed [later](#rendering-the-app).
 
 <!-- link to a "why" post --> 
 
 #### App component
 
-You may not know it, but `App` is a component.  The simplest type of React component is a [Stateless Function](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions) that accepts a `props` object and returns a [React element](https://facebook.github.io/react/blog/2015/12/18/react-components-elements-and-instances.html). For this first version of Hello Enact!, we do not accept any properties so we can safely omit that argument. Instead, we will render the greeting within a `<div>` DOM node.
+You may not know it, but `App` is a component.  The simplest type of React component is a [Stateless Function](https://reactjs.org/docs/hooks-state.html#hooks-and-function-components) that accepts a `props` object and returns a [React element](https://reactjs.org/blog/2015/12/18/react-components-elements-and-instances.html). For this first version of Hello Enact!, we do not accept any properties so we can safely omit that argument. Instead, we will render the greeting within a `<div>` DOM node.
```

Comment:
```suggestion
You may not know it, but `App` is a component.  The simplest type of React component is a [Stateless Function](https://reactjs.org/docs/components-and-props.html#function-and-class-components) that accepts a `props` object and returns a [React element](https://reactjs.org/blog/2015/12/18/react-components-elements-and-instances.html). For this first version of Hello Enact!, we do not accept any properties so we can safely omit that argument. Instead, we will render the greeting within a `<div>` DOM node.
```

Reply:
The suggested link looks better. Thank you.